data/phones/list_phones.py

```diff
@@ -63,6 +103,7 @@ def main(args: argparse.Namespace):
             f"{', '.join(_pick_examples_for_display(examples))}"
         )
     print(f"\n# unique phones: {len(phone_to_examples)}")
+    _check_ipa_phonemes(phone_to_examples, args)
 
 
 if __name__ == "__main__":
```

Comment:
Immediately below this if, add:

```python
logging.basicConfig(format="%(levelname)s: %(message)s", level="INFO")```

which enables the same logging format we use throughout. See [this](https://github.com/kylebgorman/wikipron/blob/master/wikipron/cli.py) for another example.

Reply:
Done.